### ISSUE TITLE
Showing Error after entering One Line Install. 

### ISSUE THREAD

**Issue by itsRealGray** [ChrisTitusTech/powershell-profile#16]
![Annotation 2022-11-14 210616](https://user-images.githubusercontent.com/102512869/201701181-38b4a882-7a6e-4768-954f-edb3554208e8.png)

As instructed Elevated PowerShell pasting the One Line Install line this error is popping up every time. -

Cannot bind parameter 'NewerThan'. Cannot convert value "True" to type "System.DateTime". Error: "The string
was not recognized as a valid DateTime. There is an unknown word starting at index 0."
At line:19 char:10
+          throw $_.Exception.Message
+          ~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : OperationStopped: (Cannot bind par...ng at index 0.":String) [], RuntimeExcept
   ion
    + FullyQualifiedErrorId : Cannot bind parameter 'NewerThan'. Cannot convert value "True" to type "System
   .DateTime". Error: "The string was not recognized as a valid DateTime. There is an unknown word starting
   at index 0."

**Comment by ZanzyTHEbar** [comment]
so, this is an easy fix

Use this as the entry in the script:

```powershell
#If the file does not exist, create it.
if (!(Test-Path -Path $PROFILE -PathType Leaf)) {
    try {
        # Detect Version of Powershell & Create Profile directories if they do not exist.
        if ($PSVersionTable.PSEdition -eq "Core" ) { 
            if (!(Test-Path -Path ($env:userprofile + "\Documents\Powershell"))) {
                New-Item -Path ($env:userprofile + "\Documents\Powershell") -ItemType "directory"
            }
        }
        elseif ($PSVersionTable.PSEdition -eq "Desktop") {
            if (!(Test-Path -Path ($env:userprofile + "\Documents\WindowsPowerShell"))) {
                New-Item -Path ($env:userprofile + "\Documents\WindowsPowerShell") -ItemType "directory"
            }
        }

        Invoke-RestMethod https://github.com/ChrisTitusTech/powershell-profile/raw/main/Microsoft.PowerShell_profile.ps1 -o $PROFILE
        Write-Host "The profile @ [$PROFILE] has been created."
    }
    catch {
        throw $_.Exception.Message
    }
}
```

**Comment by ZanzyTHEbar** [comment]
@ChrisTitusTech the `-ne "True"` is not required at all and breaks on most peoples setups (_including mine_)

So, instead you should remove those args and then do a boolean negation with the `!` operator instead. 

You should also lowercase the `If` to `if`. Then, your logic will work flawlessly on all machines :)

**Comment by itsRealGray** [comment]
Thanks

**Comment by ChrisTitusTech** [comment]
Merged #17
